src/core/Codend.Presentation/Controllers/EpicController.cs

```diff
@@ -0,0 +1,120 @@
+using Codend.Application.Epics.Commands.CreateEpic;
+using Codend.Application.Epics.Commands.DeleteEpic;
+using Codend.Application.Epics.Commands.UpdateEpic;
+using Codend.Contracts;
+using Codend.Contracts.Requests.Epic;
+using Codend.Domain.Core.Errors;
+using Codend.Domain.Entities;
+using Codend.Presentation.Infrastructure;
+using MediatR;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Codend.Presentation.Controllers;
+
+/// <summary>
+/// Controller for <see cref="Epic"/> commands.
+/// </summary>
+[Route("api/epic")]
+public class EpicController : ApiController
+{
+    /// <inheritdoc />
+    public EpicController(IMediator mediator) : base(mediator)
+    {
+    }
+
+    /// <summary>
+    /// Creates epic with given properties.
+    /// </summary>
+    /// <param name="request">Request with name, description and project Id.</param>
+    /// <remarks>
+    /// Sample request:
```

Comment:
Sample request is not being formatted properly without enters. Check other controllers and see where to put new lines.

![image](https://github.com/CodendDev/Codend/assets/82521883/98e771ce-8d95-49a0-8af1-b96d08422195)